pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
```

Comment:
Too much translated.
The name of the offer is VMware on OVHcloud, even for France
see https://www.ovhcloud.com/fr/hosted-private-cloud/